style/profileEditView.css

```diff
@@ -30,14 +30,15 @@ input[type=text] {
 }
 
 .form {
-  width: 780px;
+  width: 600px;
```

Comment:
Without pulling down your changes to test locally: are you sure this doesn't break the styling of the profile edit view? 

(If yes, probably makes sense to separate the two forms out into different CSS classes)

Reply:
Well, it looked...fine? but I don't know how CSS or layouts work. It's mostly that if it's 780 it gives me a horizontal scroll bar on my laptop for no discernible reason.